ansible/roles/ironic/templates/ironic-ipxe-httpd.conf.j2

```diff
@@ -0,0 +1,45 @@
+{% set server_root = '/etc/apache2' if kolla_base_distro in ['ubuntu', 'debian'] else '/etc/httpd' %}
+{% set user = 'www-data' if kolla_base_distro in ['ubuntu', 'debian'] else 'apache' %}
+
+# Apache httpd v2.4 minimal configuration
+# This can be reduced further if you remove the access log and mod_log_config
+ServerRoot "{{ server_root }}"
+ServerName {{ api_interface_address }}
+
+# Minimum modules needed
+LoadModule mpm_event_module modules/mod_mpm_event.so
+LoadModule log_config_module modules/mod_log_config.so
+LoadModule dir_module modules/mod_dir.so
+LoadModule authz_core_module modules/mod_authz_core.so
+LoadModule unixd_module modules/mod_unixd.so
+
+PidFile /tmp/httpd.pid
+
+# Comment this out if running httpd as a non root user
+User {{ user }}
+Group {{ user }}
+
+# Port to Listen on
+Listen {{ ironic_ipxe_port }}
+
+# In a basic setup httpd can only serve files from its document root
+DocumentRoot "/httpboot"
+
+# Default file to serve
+DirectoryIndex index.html
+
+LogLevel info
+ErrorLog "|/bin/cat"
+LogFormat "%h %l %u %t \"%r\" %>s %b" common
+CustomLog "|/bin/cat" common
```

Comment:
These cause logs to be shown in `docker logs`, right? The usual pattern in kolla ansible is to write logs to a file under /var/log/kolla/<service>. In this case, I guess it would be `/var/log/kolla/ironic/ironic-ipxe-error.log`.

Reply:
@jovial - This might be useful for log options and formatting: https://review.openstack.org/#/c/559375/17/ansible/roles/monasca/templates/monasca-api/wsgi-api.conf.j2

Reply:
Yeah, that shows up in `docker logs`. So outputting to a log file is preferential to something like:
https://www.fluentd.org/guides/recipes/docker-logging?

Reply:
It's the pattern that's used in kolla-ansible, for better or worse.